web-client/views/upload.jade

```diff
@@ -382,6 +391,43 @@ html
                   input(id='dashedGrayLineButton', type='checkbox', value='dashedGrayLine')
                   label(for='dashedGrayLineButton' class='sideLabel') Show Gray Edges as Dashed
 
+        //- Need to make this work.
+        div(class="data-sidebar disabled panel panel-default")
+          p(class='panel-title sidebarPanelHeader disabledOnLaunch')
+            a(id='data-sidebar' data-toggle='collapse' data-target='#dataSidebarPanel') Data
+          div(id="dataSidebarPanel" class="panel-collapse collapse")
+            div(class="panel-body")
+              div(class="dataSidebarBody")
+                p(class='sideHeader info' data-toggle='tooltip') Dataset
+                select(multiple id="data-set-select")
+                    //- Checkboxes instead of ctrl click ?
+                    //- Eventually want it to be so that the possible available ones can be detected from the network sheet
+                    //- Refresh selection upon new upload
+                    //- Add "onclick" functionality for checkbox
+                    option(value="data-set-select-barreto") Barreto 2018 wt
+                    option(value="data-set-select-dahlquistdcin5") Dahlquist 2018 dcin5
+                    option(value="data-set-select-dahlquistdgln3") Dahlquist 2018 dgln3
+                    option(value="data-set-select-dahlquistdhap4") Dahlquist 2018 dhap4
+                    option(value="data-set-select-dahlquistdzap1") Dahlquist 2018 dzap1
+                    option(value="data-set-select-dahlquistwt") Dahlquist 2018 wt
+                    option(value="data-set-select-kitagawa") Kitagawa 2002 wt
+                    option(value="data-set-select-thorsen") Thorsen 2007 wt
+                p(class='sideHeader info' data-toggle='tooltip') Timepoints
+                select(multiple id="timepoints-select")
+                    option(value="timepoint-select-10") 10
+                    option(value="timepoint-select-15") 15
+                    option(value="timepoint-select-20") 20
+                    option(value="timepoint-select-30") 30
+                    option(value="timepoint-select-40") 40
+                    option(value="timepoint-select-60") 60
+                    option(value="timepoint-select-90") 90
+                    option(value="timepoint-select-120") 120
+                    option(value="timepoint-select-1080") 1080
```

Comment:
By extension, this will change also. Again it will start empty, but when a data set is chosen, a second web service call will return the available time points for the chosen data set.